internal/debug/flags.go

```diff
@@ -167,9 +167,14 @@ var Flags = []cli.Flag{
 	traceFlag,
 }
 
+type writerCloser interface {
+	Write([]byte) (int, error)
+	Close() error
+}
+
 var (
 	glogger                *log.GlogHandler
-	logOutputF             *os.File
+	logOutputF             writerCloser
```

Comment:
I know I could have said something earlier, but.. `logOutputF` is not a great name. Can we make it a bit more descripitive? Like `logOutputFile` `logFile` or `logWriter`

Also, why not just use `io.WriteCloser`: https://pkg.go.dev/io#WriteCloser ?